setup.py

```diff
@@ -5,7 +5,6 @@
 import re
```

Comment:
Do we still need this setup.py file at all or can we get rid of it?

Reply:
No, this can go - I kept it around for the moment to check all of the metadata made it across, but I think that's all fine now.